pkgs/development/python-modules/tinycss2/default.nix

```
@@ -4,38 +4,41 @@
 , fetchFromGitHub
 , webencodings
 , pytestCheckHook
+, flit-core
 }:
 
 buildPythonPackage rec {
   pname = "tinycss2";
-  version = "1.1.0";
+  version = "1.1.1";
   disabled = pythonOlder "3.5";
```

3.5 is no longer [supported](https://github.com/Kozea/tinycss2/blob/v1.1.1/pyproject.toml#L11).

```suggestion
  disabled = pythonOlder "3.6";
```